tests/vmware/basic_net_vm_import_test.go

```diff
@@ -0,0 +1,111 @@
+package vmware_test
+
+import (
+	v2vv1 "github.com/kubevirt/vm-import-operator/pkg/apis/v2v/v1beta1"
+	"github.com/kubevirt/vm-import-operator/tests"
+	fwk "github.com/kubevirt/vm-import-operator/tests/framework"
+	. "github.com/kubevirt/vm-import-operator/tests/matchers"
+	"github.com/kubevirt/vm-import-operator/tests/utils"
+	"github.com/kubevirt/vm-import-operator/tests/vmware"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+type networkedVMImportTest struct {
+	framework *fwk.Framework
+}
+
+var _ = Describe("Networked VM import ", func() {
+	var (
+		f          = fwk.NewFrameworkOrDie("networked-vm-import", fwk.ProviderVmware)
+		secret     corev1.Secret
+		namespace  string
+		test       = networkedVMImportTest{f}
+		err        error
+	)
+
+	BeforeEach(func() {
+		namespace = f.Namespace.Name
+
+		secret, err = f.CreateVmwareSecretInNamespace(namespace)
+		if err != nil {
+			Fail("Cannot create secret: " + err.Error())
+		}
+	})
+
+	table.DescribeTable("should create started VM with pod network", func(networkType *string) {
+		vmi := utils.VirtualMachineImportCr(fwk.ProviderVmware, vmware.VM70, namespace, secret.Name, f.NsPrefix, true)
+		vmi.Spec.Source.Vmware.Mappings = &v2vv1.VmwareMappings{
+			NetworkMappings: &[]v2vv1.NetworkResourceMappingItem{
+				{Source: v2vv1.Source{Name: &vmware.VM70Network}, Type: networkType},
+			},
+		}
+		created, err := f.VMImportClient.V2vV1beta1().VirtualMachineImports(namespace).Create(&vmi)
+
+		Expect(err).NotTo(HaveOccurred())
+		Expect(created).To(BeSuccessful(f))
+
+		retrieved, _ := f.VMImportClient.V2vV1beta1().VirtualMachineImports(namespace).Get(created.Name, metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		vmBlueprint := v1.VirtualMachine{ObjectMeta: metav1.ObjectMeta{Name: retrieved.Status.TargetVMName, Namespace: namespace}}
+		Expect(vmBlueprint).To(BeRunning(f))
```

Comment:
Is that expected that the machine gets into running state without conversion? Is it the case here that the VM as a machine starts, but the system does not boot? 

It would be good to have a way of verifying whether the OS is working properly (on a separate PR, maybe in #400 - when it should actually work).